bundle

```diff
@@ -147,33 +192,23 @@ def version_bundle(bundle, channel, overrides):
                 break
         else:
             if charm.startswith("cs:"):
+                assert channel.track == "latest", f"Charmstore doesn't support numerical track {channel} for {charm}"
```

Comment:
small nit, is the problem numerical tracks, any track that isnt latest? If its the latter the error message could be clearer like "Charmstore only supports track 'latest'"

Reply:
so, the problem is Charmstore doesn't support a numerical track.  If the track is a number like `1.24` then `Channel.__str__` method will render:

```
Charmstore doesn't support numerical track 1.24/stable for jimmy-bob
```

Reply:
The thing to keep in mind is:

```channel = (track|"latest")/risk[/branch]```